src/interfaces/coral_web/src/pages/login.tsx

```diff
@@ -116,54 +122,68 @@ const LoginPage: NextPage<Props> = () => {
         </div>
 
         {hasBasicAuth && (
-          <form
-            onSubmit={handleSubmit(onSubmit)}
-            onChange={() => setErrors([])}
-            className="mt-10 flex w-full flex-col"
-          >
-            <Input
-              className="w-full"
-              label="Email"
-              placeholder="yourname@email.com"
-              type="email"
-              stackPosition="start"
-              hasError={!!formState.errors.email}
-              errorText="Please enter a valid email address"
-              {...register('email', {
-                required: true,
-                validate: (value) => simpleEmailValidation(value),
-              })}
-            />
-
-            <Input
-              className="mb-2 w-full"
-              label="Password"
-              placeholder="••••••••••••"
-              type="password"
-              actionType="revealable"
-              stackPosition="end"
-              hasError={!!formState.errors.password}
-              errorText="Please enter a valid password"
-              {...register('password', { required: true })}
-            />
-
-            {errors.map(
-              (error) =>
-                error && (
-                  <Text key={error} className="mt-4 text-danger-500 first-letter:uppercase">
-                    {error}
-                  </Text>
-                )
-            )}
-
-            <Button
-              disabled={loginStatus === 'pending' || !formState.isValid}
-              label={loginStatus === 'pending' ? 'Logging in...' : 'Log in'}
-              type="submit"
-              className="mt-10 w-full self-center md:w-fit"
-              splitIcon="arrow-right"
-            />
-          </form>
+          <>
```

Comment:
Is there any reason why we added this Fragment here? 🤔

Reply:
With the conditional render opened by `{hasBasicAuth && (`, you're required to have one parent element. Without it, the `<form>` and `<Text>` sibling elements would cause problems with ReactDOM